content/projects/gsoc/2023/projects/add-probes-to-plugin-health-score.adoc

```diff
@@ -9,36 +9,53 @@ tags:
 - plugins
 status: "Selected"
 sig: platform
-//FIXME: create bio for Jagruti Tiwari in content/_data/authors/
-student: "null_gsoc_contributor"
+student: "jagruti"
 mentors:
 - "alecharp"
 - "dheerajodha"
 - "mostafaashraf"
 links:
     idea: /projects/gsoc/2023/project-ideas/add-probes-to-plugin-health-score
-//   gitter: "some gitter channel"
-//   draft: link to the draft/final proposal
-//   meeting: link to internal section of this page with the meeting details and notes
+//   gitter: https://app.element.io/#/room/#jenkinsci_GSoC-Plugin_Health_Score:gitter.im
+//   draft: https://docs.google.com/document/d/14URAapu7svkj1pgw3yoCybwn766Net9SZ9coC4zx7yg/edit
+//   meeting: https://docs.google.com/document/d/1QcwSiAuQtoy4dGlPXgY3w8FjDzTJn-3yCv75U-OFJ04/edit
 ---
-
-//above links must be updated
-
 === Abstract
 
-(to be described)
+The project proposes to add new probes to the Plugin Health Score tool. Originally started in 2022 as a part of a GSoC project, this tool determines the health score of a plugin. Probes are the heart of the tool. To extend the work started last year, I propose adding new probes to make the tool more valuable and feature-complete.
 
 === Rationale
 
-(to be described)
+In general, probes serve two purposes within the context of this toolset:
+
+* To collect the data to compute the score of the plugin systematically. Based on the computed score, the health of the plugin can be determined.
+* Based on the score future contributors, maintainers, and adopters decide which plugin(s) they want to invest their time in.
+
+To enhance the effectiveness and usefulness of the Plugin Health Scoring System for the community, I propose to expand the number of probes used to gather data on plugin performance.
+
+While the current probes provide valuable insights, they may not capture all the necessary criteria to fully evaluate the health of a plugin. Adding more probes can provide a more comprehensive and accurate assessment of plugin health, benefiting a wider range of users.
+
 
 === Implementation
-(to be described)
+The project will be implemented in three parts:
 
-=== Office hours
+* Interaction of probes with each other
+* Probe's execution order
+* Probe's scoring
+
+Finally, the health percentage of the probe will be determined.
+
+Health percentage categories
 
-Details about the Office Hours (general or specific)
+* Low health: < Q1 (25th percentile)
+* Medium health: > Q1 (25th percentile) and < (75th percentile)
+* High health: > Q3 (75th percentile)
+
+=== Office hours
+* (General) Official weekly Jenkins office hours: Thursdays 15:00 UTC
+* (Project-based) Weekly project based office hours: Tuesdays 14:30  UTC
 
 === Links
 
-Here come some useful links
+* https://docs.google.com/document/d/1QcwSiAuQtoy4dGlPXgY3w8FjDzTJn-3yCv75U-OFJ04/edit[Meeting document]
+* https://drive.google.com/file/d/1am_kGDZbedbD0zmynIeniYeCxRfY9bZD/view?usp=drive_open[Meeting recording]
```

Comment:
That link was only for the first meeting. 
We are adding the link to each meeting recording directly on the meeting notes, so I feel that's good enough.

```suggestion
```